### DISCUSSION TITLE
Pre-Lancing Report Error - GooglePlay

### DISCUSSION THREAD

**Discussion by muriloflesch2** [airsdk/Adobe-Runtime-Support#2868] [General]
I've tried thousands of things and nothing solves this problem. I am using Air SDK 5.2.3.5 with Adobe Animate.

Google Play always return:
Install_prarse_failed_manifest_malformad

Has anyone been through this?

<img width="1244" alt="Captura de Tela 2023-10-05 às 18 59 10" src="https://github.com/airsdk/Adobe-Runtime-Support/assets/118574171/611496c1-b5c1-489f-849d-c403b6ec663e">

**Comment by marchbold** [comment]
Have you checked your manifest additions, sounds like you have an error in there?

**Reply by muriloflesch2** [reply]
I checked. The entire content of the manifest was carefully inserted based on the guidelines of ANES documentation. Animate generates the .aab file without accusing errors and I am also able to install in all my two two Android devices with no mistakes. It seems to happen only in those devices cited by GooglePlay.

**Reply by muriloflesch2** [reply]
I found the problem.

I had to remove an action from my XML that was being used by an Ane.

If someone goes through something similar, look for this cation on your XML and remove.

  --> Action Android: Name = "Android.intent.Action.view" <--
  
  Tanks  @marchbold  !

**Reply by marchbold** [reply]
Was it an activity? You might have needed to add the `android:exported="false"` flag which is a requirement for newer android api versions.
